### PR TITLE
Changed teleop url to use the same host as the proxy server

### PR DESCRIPTION
**Pull request by pablisho**

### CODE REVIEW

File: www/rocon_web_remocon/index.js [hunk 1]
```diff
@@ -327,7 +327,12 @@ function prepareWebappUrl(interaction, baseUrl) {
   // Package all the data in json format and dump it to one query string variable
   queryStringMappings = {};
   queryStringMappings['interaction_data'] = JSON.stringify(interactionData);
-    
+
+  var parser2 = document.createElement("a");
```
Comment (adamantivm) [inline www/rocon_web_remocon/index.js:331]: @pablisho can you please add a comment explaining what we're doing here?

**Comment by jihoonl** [comment]
It seems like it forces all webapps to stay together in proxy server. Is this necessary?

I have been using web remocon in proxy server(http://proxy.robotconcert.org:3040) and webapps in other webserver(http://webapp.robotconcert.org).

**Comment by adamantivm** [comment]
@jihoonl I understand.

We have the assumption that the Webapps for local access are hosted in a local Intranet computer (i.e.: localhost:8888) but for Internet access (proxy), they are available somewhere else. Is this a correct assumption? I call this Option A.
An alternative would be to consider that Webapps are always hosted in an externally-available website, like webapp.robotconcert.org, both for local and proxy access. Is this more realistic? I call this Option B.

What is the best solution for this part depends on that approach. In the first case, we need to have a default configuration in the Interactions pointing to the local access webapps (i.e.: localhost:8888) and override that value for Proxy access in the Proxy Web Remocon.

Please let us know if you think Option A or Option B is more similar to how you would use a Concert with both local and remote access.

**Comment by jihoonl** [comment]
@adamantivm I think option B is more realistic. With option B, the concert administrator can easily configure the system pointing to either local intranet webapp server or externally-available website.

**Comment by pablisho** [comment]
@jihoonl @adamantivm Regarding we are going to use option B, then this is not necessary as we will always use the address that comes from the Interaction.

**Pull request closed without merging**